src/ensembl/production/metadata/api/models/genome.py

```diff
@@ -38,6 +38,38 @@ class Genome(LoadAble, Base):
     # organism_id to organism
     organism = relationship("Organism", back_populates="genomes")
 
+    def get_public_path(self, base_path, type='all', release=None, genebuild_version=None):
```

Comment:
As I said, base_path id `not` part of the method. The API doesn't care about this, and just return relative path.